integration-tests/default_oncotree_file.txt

```diff
@@ -0,0 +1,10 @@
+SAMPLE_ID	ONCOTREE_CODE
+P-1111111-T11-IH1	MDS
+P-2222222-T22-IH2	BLL
+P-3333333-T33-IH3	PEMESO
+P-4444444-T44-IH4	MPN
+P-5555555-T55-IH5	PV
+P-6666666-T66-IH6	UNKNOWN
```

Comment:
I guess this is expected to generate "NA", "NA" in CANCER_TYPE and CANCER_TYPE_DETAILED .. but the test only fails if *all* values are NA. So one successful conversion from an oncotree code is adequate to pass the test. (I think that makes sense since we are checking the schema compatibility)